sound/soc/sof/intel/hda-loader.c

```diff
@@ -507,3 +538,16 @@ int hda_dsp_ext_man_get_cavs_config_data(struct snd_sof_dev *sdev,
 
 	return 0;
 }
+
+int hda_dsp_core_stall_icl(struct snd_sof_dev *sdev)
```

Comment:
we should be explicit in passing the core_mask to specify which core to stall here.